src/doc/unstable-book/src/language-features/asm-experimental-arch.md

```diff
@@ -48,6 +49,8 @@ This feature tracks `asm!` and `global_asm!` support for the following architect
 | M68k         | `reg_addr`     | `a[0-3]`                           | `a`                  |
 | LoongArch    | `reg`          | `$r1`, `$r[4-20]`, `$r[23,30]`     | `r`                  |
 | LoongArch    | `freg`         | `$f[0-31]`                         | `f`                  |
+| s390x        | `reg`          | `r[0-15]`                          | `r`                  |
```

Comment:
This list should exclude reserved registers (`r15` & `r11`).

Reply:
Thanks. Excluded `r15` & `r11`.